samples/server/petstore/kotlin-springboot-delegate/src/main/kotlin/org/openapitools/api/PetApi.kt

```diff
@@ -85,7 +85,7 @@ interface PetApi {
         authorizations = [Authorization(value = "petstore_auth", scopes = [AuthorizationScope(scope = "read:pets", description = "read your pets")])])
     @ApiResponses(
         value = [ApiResponse(code = 200, message = "successful operation", response = Pet::class, responseContainer = "List"),ApiResponse(code = 400, message = "Invalid status value")])
-    @GetMapping(
+    @`get`Mapping(
```

Comment:
@noordawod is this suppose to happen?

Reply:
Hmm, nope... I guess a bug in the generator, so the easiest fix is to remove `get` from the list.

Reply:
I think this already happen once and the solution was to remove `get`.
OR you can add only the words that you need.